docs/errors.rst

```diff
@@ -39,6 +39,8 @@ Add your ``SENTRY_DSN`` to your configuration file e.g. ``airflow.cfg`` under ``
 .. note::
     If this value is not provided, the SDK will try to read it from the ``SENTRY_DSN`` environment variable.
 
+You can supply `additional configuration options <https://docs.sentry.io/error-reporting/configuration/?platform=python>`__ based on the Python platform via [sentry] section.
```

Comment:
Can you also quickly note the unsupported options here, just so everything is clear?

```
UNSUPPORTED_SENTRY_OPTIONS = frozenset(
        ("integrations", "in_app_include", "in_app_exclude", "ignore_errors",
         "before_breadcrumb", "before_send", "transport")
    )
```

Reply:
Done. Add this info in doc and config templates.